.github/workflows/linter.yaml

```diff
@@ -10,15 +12,10 @@ jobs:
     env:
       CT_TARGET_BRANCH: ${{ github.event.repository.default_branch }}
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: install yq
```

Comment:
yq is not needed anymore?

Reply:
Apparently it's included by default; https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#tools (search for yq)

Reply:
Ah ok. Alright.